frontend/src/admin/components/userList/userlist.scss

```diff
@@ -0,0 +1,49 @@
+@import "src/styles/main.scss";
+@import "src/styles/mixins.scss";
+
+.userlist {
+  display: flex;
+  flex-direction: column;
+  width: 20%;
+  height: 90%;
```

Comment:
This works fine for a fixed size monitor, but will not scale well when resizing the window.
Let us have a talk about the page structure :)

Reply:
I have implemented grid